concord-ui/src/components/statstable.js

```diff
@@ -0,0 +1,193 @@
+import React, {Component} from 'react';
+
+import {
+    DataTable,
+    DataTableBody,
+    DataTableCell,
+    DataTableContent,
+    DataTableHead,
+    DataTableHeadCell,
+    DataTableRow
+} from 'rmwc/DataTable';
+import '@rmwc/data-table/data-table.css';
+
+export default class StatsTable extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = ({
+            data: [],
+            loading: true
+        })
+    }
+
+    componentWillMount() {
+        console.log("Fetching stats");
+        fetch('/api/stats', {credentials: "same-origin"})
```

Comment:
Don't think this will work in prod.

Reply:
??? More details @LaurenceTews

Reply:
Needed this for it to work in Google Chrome, found on stackoverflow..

Reply:
Will depend on our architecture. Setting credentials so 'same-origin' disables CORS errors but only works if the request URL is on the same origin as the calling script.